Makefile

```diff
@@ -507,6 +507,6 @@ vsphere-management-and-workload-cluster-e2e-test:
 	BUILD_VERSION=$(BUILD_VERSION) test/vsphere/run-tce-vsphere-management-and-workload-cluster.sh
 
 unmanaged-cluster-e2e-test:
-	cd cli/cmd/plugin/unmanaged-cluster/test/e2e && BUILD_VERSION=$(BUILD_VERSION) go test -test.v -timeout 180m
+	cd cli/cmd/plugin/unmanaged-cluster/test/e2e && BUILD_VERSION=$(BUILD_VERSION) ROOT_DIR=$(ROOT_DIR) go test -test.v -timeout 180m
```

Comment:
I think this is good 👍🏼 but I'd like to see the decouping of our unit tests and e2e tests for unmanaged-cluster happen sooner than latter since this further breaks running tests in the local directory

Reply:
Agreed. Merged it just to get rid of the red signal, but there definitely should have refactor work that needs to be done to decouple.